lib/rspec/rails/fixture_support.rb

```diff
@@ -9,6 +9,13 @@ module FixtureSupport
         include RSpec::Rails::MinitestAssertionAdapter
         include ActiveRecord::TestFixtures
 
+        if ::Rails.version.to_f >= 6.1
+          # @private return the example name for TestFixtures
+          def name
+            @example
+          end
+        end
```

Comment:
This is just moved to change the visibility right?

Reply:
Yes, that's what has fixed the issue.